manage.py

```diff
@@ -24,7 +25,7 @@ def insert_fields():
 @cli.command("run_population_job")
```

Comment:
nit: I think "update_db" would be a more concise/clear way of naming this command, but this is somewhat of personal opinion, so feel free to ignore.

Reply:
I agree.  I'm never entirely happy with the way I name things.

Reply:
Addressed with [64ed9c9](https://github.com/wikimedia/toolhunt/pull/56/commits/64ed9c916a4b149b29dda202ca9dff943c2ea1c6)